utils/helper.sh

```diff
@@ -504,3 +492,9 @@ function exit_with_error()
 	printf "\t${MESSAGE}\n"
   exit $CODE
 }
+
```

Comment:
What's the motivation of removing `check_confluent_local_service_ready` and adding `append_once`?

Reply:
`check_confluent_local_service_ready` was not being used anywhere
`append_once` prevents this from happening (after running examples more than once):

```
$ tail ksql-server.properties

# Uncomment and complete the following to enable KSQL's integration to the Confluent Schema Registry:
# ksql.schema.registry.url=http://localhost:8081
auto.offset.reset=earliest
auto.offset.reset=earliest
auto.offset.reset=earliest
auto.offset.reset=earliest
auto.offset.reset=earliest
auto.offset.reset=earliest
auto.offset.reset=earliest
```

Reply:
@brianstrauch great, thank you.

wrt `auto.offset.reset=earliest`:   @rspurgeon was also thinking to fix that as well (and might have had a PR open for that too?), cc-ing him here for additional review on that.

Reply:
This looks good to me.  I had a similiar fix but it didn't make it in so let's definitely add this one.  Thanks @brianbushree